build.gradle.kts

```diff
@@ -3,7 +3,7 @@ plugins {
     // It does not support participating in precompiled script plugins
     id("com.github.vlsi.stage-vote-release") version "1.84"
     // The Kotlin Gradle plugin was loaded multiple times in different subprojects, which is not supported and may break the build.
```

Comment:
Curious, if this comment still necessary

Reply:
Well, the comment kind of explains the reason we define `kotlin` here. The root project does not use Kotlin, so it might be tempting to remove unused bits. However, kotlin mention here is needed so it brings the plugin to the classpath once for all the subprojects (there's only one classpath anyway)